### PR TITLE
I fixed Command class to adapt trait.

### PR DESCRIPTION
**Pull request by dairyo**
***This PR is not buildable but this PR is not merged to develop. Please feel free to merge this.***

I fixed `Command` class. 

* I removed `schemaName` and `schemaVersion`.
* I replace type of `actions` and `actionResults` properties.

Related issue: #252

### CODE REVIEW

File: ThingIFSDK/ThingIFSDK/Command.swift [hunk 1]
```diff
@@ -91,61 +83,44 @@ open class Command: NSObject, NSCoding {
...
          modified: Date? = nil,
          title: String? = nil,
          commandDescription: String? = nil,
          metadata: Dictionary<String, Any>? = nil) {
-        if commandID != nil {
-            self.commandID = commandID!
-        }else {
-            self.commandID = ""
-        }
+        self.commandID = commandID ?? ""
```
Comment (yongpingchen) [inline ThingIFSDK/ThingIFSDK/Command.swift]: what about nil value for commandID?

Comment (dairyo) [reply ThingIFSDK/ThingIFSDK/Command.swift]: @yongpingchen 

It looks good. Moreover, We can make `self.command` non nil and non empty.

`Command` class represents a command created in server. The command must have command id.

I will change `commandID` argument to non optional.

**Comment by dairyo** [comment]
@yongpingchen I have fixed according to your comment. Please check this.

**Comment by yongpingchen** [comment]
LGTM

**Pull request merged**